patchwork-user-profiles.md

```diff
@@ -1,299 +0,0 @@
-# Patchwork: User profiles
```

Comment:
this is a cool document, maybe this can become a set of ssb design principles, [e.g.](https://github.com/openvocab/ovn/issues/12).